cmd/root.go

```diff
@@ -45,6 +45,7 @@ func init() {
 	rootCmd.PersistentFlags().StringP("baseline-path", "b", "", "path to baseline with issues that can be ignored")
 	rootCmd.PersistentFlags().StringP("log-level", "l", "info", "log level (trace, debug, info, warn, error, fatal)")
 	rootCmd.PersistentFlags().BoolP("verbose", "v", false, "show verbose output from scan")
+	rootCmd.PersistentFlags().Int("max-target-megabytes", 0, "maximum size for a file/blob to be scanned")
```

Comment:
This "help" message should be changed to match what is in the README per @weineran's suggestion

Reply:
@zricethezav Hi, ty for your comment. i pushed the change.
Will wait for the release its pretty impotent for me to have it so if its possible from u to have new version in couple of d will be great! ty for the tool\help\comments :)